dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jwt1.0.internal.ee-10.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
The jwt1.0.internal.ee-9.0 already had a tolerates for servlet 6 so this auto features will never be loaded I believe.  It should be removed.